api/src/test/java/org/openmrs/module/mksreports/reports/OutpatientConsultationReportManagerTest.java

```diff
@@ -171,8 +160,29 @@ public void testReport() throws Exception {
 			assertThat(allWithMalaria, is(notNullValue()));
 			assertThat(allWithMalaria.getSize(), is(3));
 		}
+	}
+	
+	@Test
+	@Ignore
+	public void test_multipleDiagnosisForTheSamePatientShouldBeCountedOnce() throws Exception {
+		Concept allDiags = inizService.getConceptFromKey("report.opdconsult.diagnosesList.concept");
```

Comment:
I realise that it would be a lot clearer to rename this config key to
```java
"report.opdconsult.diagnoses.conceptSet"
```
Could you do that and `grep` it so that it's changed everywhere it should?